docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,215 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+This procedure describes how to display Sawtooth network statistics and
+operating system metrics in Grafana, using Docker, InfluxDB, and Telegraf.
+
+.. note::
+
+   This procedure is for a Linux (Ubuntu) environment. For a Sawtooth network
+   in Docker containers, additional steps are needed to change the
+   configuration of the validator containers. These steps are not described in
+   this procedure.
+
+Requirements
+============
+
+This guide assumes Docker Engine is installed and one or more sawtooth
+validator nodes have been configured.
+
+Download `sawtooth-core <https://github.com/hyperledger/sawtooth-core>`_
+from the Hyperledger GitHub repository.
+
+.. code-block:: console
+
+   $ git clone https://github.com/hyperledger/sawtooth-core.git
+
+InfluxDB
+========
+
+#. First, pull the InfluxDB Docker image from Docker Hub.
+
+   .. code-block:: console
+
+     $ docker pull influxdb
+
+#. Create an InfluxDB data directory to provide persistent storage on the local
+   file system.
+
+   .. note::
+
+      Depending on the host operating system /var/lib/influx-data may not be
+      appropriate, modify this path accordingly.
+
+   .. code-block:: console
+
+     $ sudo mkdir /var/lib/influx-data
+
+#. Next, start an InfluxDB Docker container. This command uses the persistent
+   storage on the local file system and exposes port 8086 to the network. It
+   also configures the database name and the ``admin`` and ``lrdata`` users for
+   InfluxDB.
+
+   .. code-block:: console
+
+     $ docker run -d -p 8086:8086 -v /var/lib/influx-data:/var/lib/influxdb \
+       -e INFLUXDB_DB=metrics -e INFLUXDB_HTTP_AUTH_ENABLED=true \
+       -e INFLUXDB_ADMIN_USER=admin -e INFLUXDB_ADMIN_PASSWORD='{admin-pw}' \
+       -e INFLUXDB_USER=lrdata -e INFLUXDB_USER_PASSWORD='{lrdata-pw}' \
+       --name sawtooth-stats-influxdb influxdb
+
+   .. note::
+
+      Also, remember to properly escape any special characters, such as
+      ``,@!$``.
+
+Grafana
+=======
+
+#. First, build the Grafana Docker image from the Dockerfile included in the
+   sawtooth-core repository.
+
+   .. code-block:: console
+
+     $ cd sawtooth-core/docker
+     $ docker build . -f grafana/sawtooth-stats-grafana \
+       -t sawtooth-stats-grafana
+
+#. Run the Grafana container.
+
+   .. code-block:: console
+
+     $ docker run -d -p 3000:3000 --name sawtooth-stats-grafana \
+       sawtooth-stats-grafana
+
+#. Next, open the Grafana web page, at ``http://{host}:3000``.
+
+   .. note::
+      Be sure to replace {host} with the IP or Fully Qualified Domain Name
+      (FQDN) of the system running the Grafana Docker container.
+
+#. Log in as user ``admin`` with the password ``admin``.
+
+#. Change the admin password.
+
+   Change the admin password: First, click on the Grafana spiral icon at the
```

Comment:
Delete the duplicate "Change the admin password" and make this one paragraph (no extra blank line).